src/lib.rs

```diff
@@ -144,11 +144,14 @@
 //!
 //! | Feature name  | Description | Available PubNub APIs |
 //! | :------------ | :---------- | :------------- |
-//! | `full`        | Enables all non-conflicting features | Configuration, Publish, Access Manager, Parse Token |
-//! | `default`     | Enables default features: `publish`, `serde`, `reqwest`, `aescbc`, `std` | Configuration, Publish |
+//! | `full`        | Enables all non-conflicting features | Configuration, Publish, Subscribe, Access Manager, Parse Token, Presence |
+//! | `default`     | Enables default features: `publish`, `subscribe`, `serde`, `reqwest`, `aescbc`, `std` | Configuration, Publish, Subscribe |
```

Comment:
@Xavrax in `Cargo.toml` I've added `presence` as part of the default. It shouldn't be in the default list?
Also, we are missing `access` which is in `Cargo.toml` rn.

Reply:
I've remove presence from default (because it's not default in portal)

Reply:
I see access in cargo `full` and lib.rs `table`